### ISSUE TITLE
Clone Error

### ISSUE THREAD

**Issue by Seo-Bangkok** [sungik-choi/gatsby-starter-apple#43]
`Cloning into 'gatsby-starter-apple'...
remote: Enumerating objects: 103, done.
remote: Counting objects: 100% (103/103), done.
remote: Compressing objects: 100% (89/89), done.
remote: Total 103 (delta 0), reused 74 (delta 0), pack-reused 0
success Created starter directory layout
info Installing packages...

npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: gatsby-remark-vscode@3.3.0
npm ERR! Found: gatsby@4.5.4
npm ERR! node_modules/gatsby
npm ERR!   gatsby@"^4.5.4" from the root project
npm ERR!   peer gatsby@"^4.0.0-next" from babel-plugin-remove-graphql-queries@4.5.2
npm ERR!   node_modules/babel-plugin-remove-graphql-queries
npm ERR!     babel-plugin-remove-graphql-queries@"^4.5.2" from gatsby@4.5.4
npm ERR!     babel-plugin-remove-graphql-queries@"^4.4.0" from gatsby-plugin-image@2.4.0
npm ERR!     node_modules/gatsby-plugin-image
npm ERR!       gatsby-plugin-image@"^2.4.0" from the root project
npm ERR!     1 more (gatsby-plugin-typescript)
npm ERR!   19 more (gatsby-plugin-alias-imports, gatsby-plugin-feed, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer gatsby@"^2.12.1 || ^3.1.1" from gatsby-remark-vscode@3.3.0
npm ERR! node_modules/gatsby-remark-vscode
npm ERR!   gatsby-remark-vscode@"^3.3.0" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: gatsby@3.14.6
npm ERR! node_modules/gatsby
npm ERR!   peer gatsby@"^2.12.1 || ^3.1.1" from gatsby-remark-vscode@3.3.0
npm ERR!   node_modules/gatsby-remark-vscode
npm ERR!     gatsby-remark-vscode@"^3.3.0" from the root project
npm ERR!`

**Comment by sungik-choi** [comment]
Would you like to try `npm install --legacy-peer-deps` or `--force` option?

**Comment by cobyiv** [comment]
I'm getting similar issue when attempting to build off Gatsby Cloud. Below is taken from Gatsby Cloud Raw Log. Must I build outside of Gatsby Cloud in order to correct gatsby-remark-vscode module issue? 

```
13:26:43 PM:
Cloning into '/usr/src/app/www'...

13:26:43 PM:
Using node version from .nvmrc: lts/*

13:26:43 PM:
Installing node version lts/*

13:26:44 PM:
Downloading and installing node v16.17.0...

13:26:47 PM:
Now using node v16.17.0 (npm v8.15.0)

13:26:54 PM:
npm

13:26:54 PM:
 ERR! code ERESOLVE

13:26:54 PM:
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: gatsby-remark-vscode@3.3.0
npm ERR! Found: gatsby@4.5.4
npm ERR! node_modules/gatsby
npm ERR!   gatsby@"^4.5.4" from the root project
npm ERR!   peer gatsby@"^4.0.0-next" from babel-plugin-remove-graphql-queries@4.5.2
npm ERR!   node_modules/babel-plugin-remove-graphql-queries
npm ERR!     babel-plugin-remove-graphql-queries@"^4.5.2" from gatsby@4.5.4
npm ERR!     babel-plugin-remove-graphql-queries@"^4.4.0" from gatsby-plugin-image@2.4.0
npm ERR!     node_modules/gatsby-plugin-image
npm ERR!       gatsby-plugin-image@"^2.4.0" from the root project
npm ERR!     1 more (gatsby-plugin-typescript)
npm ERR!   19 more (gatsby-plugin-alias-imports, gatsby-plugin-feed, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer gatsby@"^2.12.1 || ^3.1.1" from gatsby-remark-vscode@3.3.0
npm ERR! node_modules/gatsby-remark-vscode
npm ERR!   gatsby-remark-vscode@"^3.3.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: gatsby@3.14.6
npm ERR! node_modules/gatsby
npm ERR!   peer gatsby@"^2.12.1 || ^3.1.1" from gatsby-remark-vscode@3.3.0
npm ERR!   node_modules/gatsby-remark-vscode
npm ERR!     gatsby-remark-vscode@"^3.3.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/gatsby/.npm/eresolve-report.txt for a full report.

.
.
.
.
.
13:27:01 PM:
NPM ci --unsafe-perm --prefer-offline --no-audit --registry=https://registry.npmjs.org/ attempt 3 of 3...

13:27:04 PM:
npm

13:27:04 PM:
 ERR! code ERESOLVE

13:27:04 PM:
npm ERR! ERESOLVE could not resolve
npm 

13:27:04 PM:
ERR! 
npm ERR! While resolving: gatsby-remark-vscode@3.3.0
npm ERR! Found: gatsby@4.5.4
npm ERR! node_modules/gatsby
npm ERR!   gatsby@"^4.5.4" from the root project
npm ERR!   peer gatsby@"^4.0.0-next" from babel-plugin-remove-graphql-queries@4.5.2
npm ERR!   node_modules/babel-plugin-remove-graphql-queries
npm ERR!     babel-plugin-remove-graphql-queries@"^4.5.2" from gatsby@4.5.4
npm ERR!     babel-plugin-remove-graphql-queries@"^4.4.0" from gatsby-plugin-image@2.4.0
npm ERR!     node_modules/gatsby-plugin-image
npm ERR!       gatsby-plugin-image@"^2.4.0" from the root project
npm ERR!     1 more (gatsby-plugin-typescript)
npm ERR!   19 more (gatsby-plugin-alias-imports, gatsby-plugin-feed, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer gatsby@"^2.12.1 || ^3.1.1" from gatsby-remark-vscode@3.3.0
npm ERR! node_modules/gatsby-remark-vscode
npm ERR!   gatsby-remark-vscode@"^3.3.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: gatsby@3.14.6
npm ERR! node_modules/gatsby
npm ERR!   peer gatsby@"^2.12.1 || ^3.1.1" from gatsby-remark-vscode@3.3.0
npm ERR!   node_modules/gatsby-remark-vscode
npm ERR!     gatsby-remark-vscode@"^3.3.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/gatsby/.npm/eresolve-report.txt for a full report.

13:27:04 PM:
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gatsby/.npm/_logs/2022-09-18T20_27_02_404Z-debug-0.log

13:27:04 PM:
NPM ci --unsafe-perm --prefer-offline --no-audit --registry=https://registry.npmjs.org/ failure - Command failed with exit code 1 (EPERM): npm ci --unsafe-perm --prefer-offline --no-audit --registry=https://registry.npmjs.org/

13:27:04 PM:
ERROR Failed to compile: Error: Command failed with exit code 1 (EPERM): npm ci --unsafe-perm --prefer-offline --no-audit --registry=https://registry.npmjs.org/
```

**Comment by sungik-choi** [comment]
> @cobyiv 

Would you like to try npm install --legacy-peer-deps or --force option?

**Comment by sungik-choi** [comment]
see also: https://www.gatsbyjs.com/docs/how-to/cloud/npm-7-peer-dependencies/

**Comment by sungik-choi** [comment]
Fixed by #45
